src/app/pages/NFTByFtxId/HeroleteItem/HeroleteItem.module.scss

```diff
@@ -2,14 +2,26 @@
 
 .heroleteItem {
   display: flex;
-  justify-content: space-around;
+  flex-direction: column;
   margin: 0.5rem 0;
   text-align: center;
 
-  &___info {
+  > div {
```

Comment:
can we use a class instead of child selector here? classes are more maintainable as we can add elements between and won't break our styles

Reply:
I agree

Reply:
Done!